### ISSUE TITLE
Selectors specificity should match levels

### ISSUE THREAD

**Issue by ffoodd** [ffoodd/a11y.css#244]
In this [accessibility fails test page](http://aduggin.github.io/accessibility-fails/), the "Image has empty alt and non-empty title" test for example matches two a11y.css selectors:
* `img[alt][title]` is an advice;
* `img[alt=""]` is a warning.

But since the advice's selector's specificity is higher, image's outline is green.

I need to think about increasing specificity for warnings and errors.

**Comment by 7studio** [comment]
I don't know if this solution can work properly for all your selectors but you can double the attribute selector like this `img[alt=""][alt=""]` to increase the specificity :grin:
Test: http://codepen.io/7studio/pen/KWEVqB

Obviously, you can use `img[alt][alt=""]` which is more understandable :joy:

**Comment by ffoodd** [comment]
Yeah I also found a mixin somewhere to duplicate selector as much as needed (something like `@include specificity(101);`) but that seems heavy to use in a11y.css -_-

For now I might try to use `!important` on errors, and not on other levels… Just an idea :)

**Comment by ffoodd** [comment]
Another crazy idea (for the records) consists in fattening selectors up with some unusual prefixes:
* starting with `:root` — and repeating `:root:root:root` — or to be really crazy `html:root > head + body`
* using and repeating something like `:not(#z)` as much as needed…

So many ways to increase specificity on very generic selectors :nerd_face: 

Source: https://philipwalton.com/articles/do-we-actually-need-specificity-in-css/

Still not sure about specificity being confusing though. Probably not something to be concerned about.

**Comment by ffoodd** [comment]
As a conclusion (because I'm closing this one 🙈) I guess that anyone annoyed with green errors as in the example may use error/warning only files.

Obviously.
